src/components/BounceRulesContainer/index.jsx

```diff
@@ -13,43 +14,41 @@ export default class BounceRules extends React.Component {
       selectedRule: {},
       rules: [],
       pageIndex: 1,
-      pageInterval: 10,
+      pageInterval: 10
     };
 
     this.updateSearchToken = this.updateSearchToken.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
   }
 
   async componentDidMount() {
-    const { data, status } = await axios.get(
-      "http://localhost:3000/bounce_rules"
-    );
+    const { data, status } = await listRules();
```

Comment:
Cool thanks for abstracting this out. hopefully this will help to make testing a bit more easier down the road when you implement some unit tests